src/Extensions/Abstractions/EventSourcesEventIds.cs

```diff
@@ -0,0 +1,56 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+namespace Microsoft.Omex.Extensions.Abstractions
+{
+	/// <summary>Event ids enum</summary>
+	public enum EventSourcesEventIds
+	{
+		/// <summary>Event Id for logging general info message</summary>
+		LogInfoEventId = 1,
+
+
+		/// <summary>Event Id for logging error message</summary>
+		LogErrorEventId = 2,
+
+
+		/// <summary>Event Id for logging warning message</summary>
+		LogWarningEventId = 3,
+
+
+		/// <summary>Event Id for logging verbose message</summary>
+		LogVerboseEventId = 4,
+
+
+		/// <summary>Event Id for logging spam message</summary>
+		LogSpamEventId = 5,
+
+
+		/// <summary>Event Id for logging timed scopes</summary>
+		LogTimedScopeEventId = 6,
+
+
+		/// <summary>Event Id for logging timed scopes traces</summary>
+		LogTimedScopeTestContextEventId = 7,
+
+
+		/// <summary>Event Id for service type registered</summary>
+		ServiceTypeRegisteredEventId = 11,
```

Comment:
Are we sure we'll never want more than 10 logging eventId? Consider starting the serviceEventId at 100

Reply:
Could you please explain more, how would it help? This value already used, so I think we need to switch to this version of enum and only then make changes